plugins/outputs/zabbix/zabbix.go

```diff
@@ -0,0 +1,164 @@
+package zabbix
```

Comment:
Please add the go-generator comment to automatically embed the sample.conf into the README on build
```suggestion
//go:generate ../../../tools/readme_config_includer/generator
package zabbix
```